### ISSUE TITLE
Drush alias

### ISSUE THREAD

**Issue by Decipher** [LionsAd/drupal_ti#29]
It'd be really nice if a Drush alias was created and 'drush use'd so that interactions with the Drupal site could be done without the need of navigating the filesystem.

**Comment by LionsAd** [comment]
I still think this makes sense regardless of this, especially using the behat extension with its alias feature.

**Comment by Decipher** [comment]
I agree it makes sense, although I'm not familiar with the alias feature of the behat extension, will look into it.

Ideally I was hoping to take a super simple approach to the Alias, by piping the output of 'drush sa @self' from within the site directory out to an alias file, which would get you halfway there, but the assigned alias would be 'self', which is relatively un-descriptive.

The other reason I didn't go down that track is just my lack of understanding on how best to fork and test a composer based package with Travis. I did try forking, and updating the namespace for the drupal_ti, but it appears it's not as easy as that, so any advice would be appreciated.

**Comment by LionsAd** [comment]
Forking is really simple:

Just fork and then instead of:

```
composer require lionsad/drupal_ti
```

you do something like:

```
- composer self-update

- git clone yourrepo/drupal_ti
- mkdir -p "$HOME/.composer/vendor/bin"
- ln -sf $(pwd)/drupal_ti/drupal-ti "$HOME/.composer/vendor/bin"
```

drupal_ti can work from everwhere.

That is the trick I use to have drupal_ti test 'itself'.

**Comment by Decipher** [comment]
Thanks, that's clearly obvious. I'll give it a spin shortly.
